langkit/response_hallucination.py

```diff
@@ -57,7 +57,7 @@ def to_summary_dict(self):
         }
 
 
-class ConsistencyCheker:
+class ConsistencyChecker:
```

Comment:
This will break the below code, you need to update all references to the old class name, e.g. see lines 241 and 252 in this file.

Reply:
the CI checks catch it: https://github.com/whylabs/langkit/actions/runs/6657337260/job/18091837714?pr=172#step:10:35